salt/modules/zypperpkg.py

```diff
@@ -1617,12 +1617,12 @@ def upgrade(refresh=True,
             dryrun=False,
             dist_upgrade=False,
             fromrepo=None,
-            novendorchange=False,
+            novendorchange=True,
             skip_verify=False,
             no_recommends=False,
             root=None,
             **kwargs):  # pylint: disable=unused-argument
-    '''
+    """
```

Comment:
@mseidl this change (the double quotes) must be removed here since it's breaking the doc string.
```suggestion
    '''
```